src/sentry/static/sentry/app/views/groupDetails/organization/groupTagValues.jsx

```diff
@@ -1,9 +1,20 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import SentryTypes from 'app/sentryTypes';
+
 import GroupTagValues from '../shared/groupTagValues';
 
-export default class OrganizationGroupTagValues extends React.Component {
+class OrganizationGroupTagValues extends React.Component {
+  static propTypes = {
+    group: SentryTypes.Group.isRequired,
+    location: PropTypes.object,
+  };
+
   render() {
-    const query = {};
+    // Clone so we don't have props mutate in place.
+    const query = {...this.props.location.query};
```

Comment:
Are there any query params we don't want to keep?

Reply:
We only use the cursor key right now. I could extract only that key.

Reply:
Let's keep this (we'd probably want to preserve global selection header keys too, no?)